db/schema.rb

```diff
@@ -0,0 +1,22 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.1].define(version: 2023_10_23_125649) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "greetings", force: :cascade do |t|
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
```

Comment:
- [ ] Good job working on your app so far 👏. However, at this moment, we won't be able to store a greeting in the database and test that your app is working correctly because there's no column for a `greeting value`. So please fix that by using migration to add a column of type `string` where we can store a greeting.  :ok_hand: 😊
- [ ] And last but not least, please, before submitting for a second review, ensure your entire app is working as expected. 😊